src/hono-base.ts

```diff
@@ -372,8 +372,8 @@ class Hono<E extends Env = Env, S extends Schema = {}, BasePath extends string =
   #addRoute(method: string, path: string, handler: H) {
     method = method.toUpperCase()
     path = mergePath(this._basePath, path)
-    const r: RouterRoute = { path, method, handler }
-    this.router.add(method, path, [handler, r])
+    const r: RouterRoute = { path: path, method: method, handler: handler }
```

Comment:
```suggestion
    const r: RouterRoute = { path, method, handler }
```

I think this section should be written in shorthand property.
(https://github.com/honojs/hono/pull/3640)